### PR TITLE
Fixed undefined variable (bsc#1040944)

### PR DESCRIPTION
**Pull request by lslezak**
## Changes

- Define `ret` (was accidentally removed during the previous cleanup)
- The `:force` check box does not need the `Opt(:notify)` option as there is nothing changed in the UI after clicking it
- Use `Wizard.CloseDialog` (as the correct counterpart for the used `Wizard.OpenDialog`)
- Extended the test suite
- Call `rpm -qa | sort` at Travis for easier debugging. I think it would make sense to add it to all modules because it's not trivial to find out which package versions are included in the latest Docker image.
- Switched from autotools to Rake 
- 3.2.2

### Note

Originally I thought I would add the `rpm` call into the shared `yast-travis-ruby` script. But that would make default Travis logs too verbose. The advantage of the `before_install` section is that the output of these commands is folded by default. See the Travis output for this PR.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ services:
 
 before_install:
   - docker build -t yast-s390-image .
+  # list the installed packages (just for easier debugging)
+  - docker run --rm -it yast-s390-image rpm -qa | sort
```
Comment (jreidinger) [inline .travis.yml]: should it be left here?

File: Rakefile [hunk 2]
```diff
@@ -3,6 +3,8 @@ require "yast/rake"
 # do nothing in osc:build as it can be builds exclusivelly on s390
 Rake::Task["osc:build"].clear_actions if `uname --machine` !~ /s390/
 
+Yast::Tasks.submit_to((ENV["YAST_SUBMIT"] || :factory).to_sym)
```
Comment (jreidinger) [inline Rakefile]: why needed? if it left empty, it will be easier for everyone, especially when branching. As this is default value.

Comment (lslezak) [inline Rakefile]: I thought to use it in Jenkins, but for branching the defaults will be better.

File: .travis.yml (hunk 1 above)
Comment (lslezak) [inline .travis.yml]: `--rm` removes the result of the command, `rpm -qa` does not have any side effects so it's not strictly needed. But'd keep it for consistency, if we add something more complex later then we do not forget about it.

File: Rakefile (hunk 2 above)
Comment (jreidinger) [inline Rakefile]: well, it is basically default. So jenkins work fine also without this line as YAST_SUBMIT is by default respected if not overwritten by submit_to

**Pull request merged**